virtme_ng/run.py

```diff
@@ -49,20 +50,38 @@ def check_call_cmd(command, quiet=False, dry_run=False):
         stdout_fd = process.stdout.fileno()
         stderr_fd = process.stderr.fileno()
 
+        stdout_open = True
+        stderr_open = True
+
         # Use select to poll for new data in the file descriptors
-        while process.poll() is None:
-            ready_to_read, _, _ = select([stdout_fd, stderr_fd], [], [], 1)
-            for file in ready_to_read:
-                if file == stdout_fd:
+        while stdout_open or stderr_open:
```

Comment:
I guess we could use a `while True` here if you validate these conditions just after, but that's a detail, no need to change :)

Reply:
Oh that's right, let me change that to a `while True`, thanks for noticing!